docker/indexer/indexer.go

```diff
@@ -37,6 +37,7 @@ var (
 	worker        = flag.Bool("worker", false, "makes this a worker node reading from pubsub to process the data")
 	pubsubTopic   = flag.String("topic", "", "sets the pubsub topic to publish to or to read from")
 	subName       = flag.String("subscription", "", "sets the pubsub subscription name for workers")
+	subMessages   = flag.Int("messages", 1, "pubsub outstanding messages")
```

Comment:
Why do we need this to be configurable here? Does 1 not work well in general?

Reply:
It avoids recompilation if it needs to be readjusted. I used it for local testing on a VM instead of a K8s cluster. Furthermore, if we see to little CPU utilization on workers it's a just a flag change.